data/presets/presets/amenity/toilets/disposal/portapotty.json

```diff
@@ -0,0 +1,31 @@
+{
+    "icon": "tnp-2009541",
+    "fields": [
+        "toilets/disposal",
+        "{amenity/toilets}"
+    ],
+    "moreFields": [
+        "{amenity/toilets}"
+    ],
+    "geometry": [
+        "point",
+        "area",
+        "vertex"
+    ],
+    "terms": [
+        "restroom",
+        "privy",
+        "portable toilet",
+        "chemical toilet"
+    ],
+    "tags": {
+        "amenity": "toilets",
+        "toilets:disposal": "chemical",
+        "portable": "yes"
+    },
+    "reference": {
+        "key": "toilets:disposal",
+        "value": "chemical"
+    },
+    "name": "Porta Potty"
```

Comment:
How about calling this by the generic term "Portable Toilet"? There are loads of slang and regional names we can put in `terms` later.